spec/ddtrace/contrib/presto/client_spec.rb

```diff
@@ -60,11 +52,16 @@ def suppress_warnings
       Datadog.registry[:presto].reset_configuration!
       example.run
       Datadog.registry[:presto].reset_configuration!
+      Datadog.configuration.reset!
     end
   end
 
   context 'when the tracer is disabled' do
-    before(:each) { tracer.enabled = false }
+    before do
+      Datadog.configure do |c|
+        c.tracer.enabled = false
```

Comment:
Should we add an `after` to reset this? If we're setting a test tracer explicitly before each test, resetting this value technically will have no effect (hence why this might be working right now), but the `enabled` setting will likely persist on the global configuration unless it too is totally reset/replaced.

Reply:
This new clean up code introduced in this PR is saving this test from failure: https://github.com/DataDog/dd-trace-rb/blob/b5b13c221917cde4e018001afc1c4a45a37b92f2/spec/ddtrace/contrib/support/spec_helper.rb#L22

I do agree that it should reset it, I'll add explicit code in here for that.